### PR TITLE
[BUGFIX] partially satisfy psalm for tests/Unit/HtmlProcessor/HtmlPrunerTest.php

### PR DESCRIPTION
**Pull request by SignpostMarv**

### CODE REVIEW

File: tests/Unit/HtmlProcessor/HtmlPrunerTest.php [hunk 1]
```diff
@@ -611,7 +635,7 @@ private static function assertSubstringCount(int $expectedCount, string $haystac
         self::assertSame(
             $expectedCount,
             \substr_count($haystack, $needle),
-            'asserting \'' . $haystack . '\' contains ' . $expectedCount . ' instance(s) of "' . $needle . '"'
+            'asserting \'' . $haystack . '\' contains ' . (string)$expectedCount . ' instance(s) of "' . $needle . '"'
```
Comment (JakeQZ) [inline tests/Unit/HtmlProcessor/HtmlPrunerTest.php]: This is a positive change, as I've already said :)

**Comment by SignpostMarv** [comment]
should note I'm cherry-picking here and leaving it up to travis to catch any `composer run ci` errors.

**Comment by JakeQZ** [comment]
> it up to travis to catch any `composer run ci` errors.

That's what it's there for.  It won't (or shouldn't) allow us to commit a broken build :)

**Comment by oliverklee** [comment]
> That's what it's there for. It won't (or shouldn't) allow us to commit a broken build :)

There is a possible edge case: If two commits from separate PRs clash with each other without causing a Git conflict and the second PR is merged without rebasing and running Travis CI again, the master build will fail. I've had this every now an then in some projects.

**Comment by JakeQZ** [comment]
> I mean as opposed to redoing the psalm commits from scratch :P

How we're doing things right now seems to be working well.  The PRs are coming through **_thin_** and fast - at lightning speed! :))

**Pull request merged**